### PR TITLE
NR-118734 docs for entity signals component

### PR DESCRIPTION
**Pull request by Marty42Martinez**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

This PR will add docs for the recently released Distributed tracing Entity Signals component that can be found on the APM Summary page.

### CODE REVIEW

**Comment by Marty42Martinez** [comment]
gatsby clean

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import signalInsightHorizontalView from 'images/distributed-tracing_entity-signal-insight-horizontal.webp'
+import signalInsightWithLegend from 'images/distributed-tracing_single-entity-card-with-legend.webp'
+import signalSummariesWithAllClear from 'images/distributed-tracing_signal-summaries-with-all-clear-example.webp'
+import noTraceDataPattern from 'images/distributed-tracing_entity-summaries-when-no-trace-data-present.webp'
+
+
+[Distributed tracing data](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#trace-structure) represents how the performance of entities across your distributed system impact each other.
+Insights from this data are surfaced with respect to individual entities you view, showing when their performance might be impacted by other traced entities.
+
+The Distributed trace insights component surfaces three types of performance signals from related traced entities.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:20]:
```suggestion
The distributed trace insights component surfaces three types of performance signals from related traced entities.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+* Call counts: Entities making a significantly increased number of calls through the entity you are viewing. This increase impacts the throughput on your entity.
+* Exclusive duration: Entities called down to by the entity you are viewing, which are contributing significantly more latency. The latency contributed by an entity is the wall-clock time that it had one or more processes running but was **not** making any external or database calls.
+* Error rates: Entities called down to by the entity you are viewing, which have more processes ending in errors.
+
+The related trace entity signals view is focused on helping you identify significant performance changes that might be impacting the entity you are viewing.
+Traced entities are only listed if their impact on performance is relatively significant to the one you are viewing, and if that impact has increased with respect to the selected and previous time ranges.
+To maintain this focus, other entities that participate in traces with the one you are viewing are not shown here if their performance impact is relatively consistent or smaller.
+
+## Summary of trace Insights
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:30]:
```suggestion
## Summary of trace insights
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+When you select a card from the initial view, you're shown a list of entities with a significant performance impact and change for the signal type you selected. Additionally, a control switch will appear above the list that allows you to toggle between the lists of each signal. In the event that you toggle to a signal with zero available insights, you will see an all clear message in place of the list.
+
+<img
+  title="Signal insight breakdown"
+  alt="Single signal insight with Error rate as the selected signal"
+  src={signalInsightWithLegend}
+/>
+
+Each item in the list contains:
+1. **Call path direction**: `Upstream | Downstream` followed by the average number of entities (hops) in the call path
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:62]:
```suggestion
1. **Call path direction**: `Upstream | Downstream` followed by the average number of entities (hops) in the call path.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    * Upstream: Insight data is generated from call paths from the listed entity to your entity.
+    * Downstream: Insight data is generated from call paths which follow calls from your entity and include the listed entity.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:64]:
```suggestion
    * `Upstream`: Insight data is generated from call paths from the listed entity to your entity.
    * `Downstream`: Insight data is generated from call paths that follow calls from your entity and include the listed entity.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+2. Selected Signal insights:Each item displays its performance for the selected signal at the top, including a trend arrow (up or down; when data is available), signal value, & percent change of the signal value (when data is available). \
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:65]:
```suggestion
2. **Selected signal insights**: Each item displays its performance for the selected signal at the top, including a trend arrow (up or down when data is available), signal value, and percent change of the signal value (when data is available).
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+Percent change: The time window for this value is defined by a combination of the time picker & Compare with dropdown values. Percent change will not be shown when previous tracing data is unavailable; this generally occurs when the time window extends past 3 days before the present time.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:66]:
```suggestion
3. **Percent change**: The time window for this value is defined by a combination of the time picker and comparison with dropdown values. Percent change will not be shown when previous tracing data is unavailable; this generally occurs when the time window extends past 3 days before the present time.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+3. Entity health icon: This icon will change status color based on the `Alert Severity` that the listed entity reports.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:67]:
```suggestion
4. **Entity health icon**: This icon will change status color based on the `Alert Severity` that the listed entity reports.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+4. Entity: The name of the entity reporting insights that impact your entity for the selected signal.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:68]:
```suggestion
5. **Entity**: The name of the entity reporting insights that impact your entity for the selected signal.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+5. Non-selected Signals: Each item will also report the signal value and percent change (when available) for the 3 signals that are not actively selected. Those signals are defined as:
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:69]:
```suggestion
6. **Non-selected signals**: Each item will also report the signal value and percent change (when available) for the three signals that are not actively selected. Those signals are defined as:
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+    * Count: The number of traced calls to or from the listed entity, when it's in a call path with your entity.
+    * Error rate: The percent of calls to the listed entity that returned errors (had span errors on their entry spans).
+    * Duration: The average duration of calls that include your entity and the listed entity in the call path.
+    * Exclusive: The average exclusive duration of calls that include your entity and the listed entity in the call path.
+6. Sparkline chart: Includes 2 timeseries of the Selected Signal value, this chart has mouse hover interactions that sync with all timeseries charts on the page. The timeseries in color covers the time window defined by the time picker while the gray timeseries presents previous data (covering the same duration) that is used for comparison.
+7. View Trace button: Clicking on this button will open a pane to the [trace details UI](/docs/distributed-tracing/ui-data/trace-details/). You will view a trace that contains your entity, the listed entity, and similar values for the Selected Signal; additionally the waterfall will render highlighted rows for spans connected to the listed entity.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:75]:
```suggestion
7. **Sparkline chart**: Includes 2 timeseries of the Selected Signal value, this chart has mouse hover interactions that sync with all timeseries charts on the page. The timeseries in color covers the time window defined by the time picker while the gray timeseries presents previous data (covering the same duration) that is used for comparison.
8. **View Trace button**: Clicking on this button will open a pane to the [trace details UI](/docs/distributed-tracing/ui-data/trace-details/). You will view a trace that contains your entity, the listed entity, and similar values for the Selected Signal. Additionally, the waterfall will render highlighted rows for spans connected to the listed entity.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+Lists have an initial limit of 20 entities; in the event that there are more than 20 entities with a significant performance impact a `Show More` button appears at the bottom of the list.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:77]:
```suggestion
Lists have an initial limit of 20 entities; in the event that there are more than 20 entities with a significant performance impact a **Show More** button appears at the bottom of the list.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+## Compare with options
+By default, performance for the selected time in the time picker is compared to immediately preceding time, over the same duration. For example, when viewing the last 30 minutes in the time picker, performance is compared to the 30 minutes just before that time. This also allows you to select a spike in one of the other charts on the page, and see if this correlates with a significant performance impact from a related trace entity.
+
+You may override the default compare with behavior by changing the value of the Compare with dropdown at the top of the [APM Summary](/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics/) page. Modifying this selection will update the end of the comparison time window (without changing duration) used to calculate percent change of signal values in the following ways:
+* none: Comparison time window will end at the beginning of the time picker window.
+* yesterday: Comparison time window will end one day before the beginning of the time picker window.
+* last week: Comparison time window will end 7 days before the beginning of the time picker window.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:85]:
```suggestion
* **None**: The comparison time window will end at the beginning of the time picker window.
* **Yesterday**: The comparison time window will end one day before the beginning of the time picker window.
* **Last week**: The comparison time window will end 7 days before the beginning of the time picker window.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+If there is no tracing data preserved in the comparison window, the Distributed trace insights subheader will not contain a "compared with" clause.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:87]:
```suggestion
If there is no tracing data preserved in the comparison window, the distributed trace insights subheader will not contain a "compared with" clause.
```

File: src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+
+## When no tracing data is present
+If your entity is not reporting trace data, the entity insights component will present a no data view that provides links to our docs and a Guided Install solution.
```
Comment (ally-sassman) [inline src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx:90]:
```suggestion
If your entity is not reporting trace data, the entity insights component will present a "no data" view which includes links to our docs and a Guided Install setup.
```

**Review by ally-sassman — CHANGES_REQUESTED** [review]

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**